include/aspect/utilities.h

```diff
@@ -44,7 +44,7 @@ namespace aspect
    */
   namespace Utilities
   {
-    using namespace dealii;
+    using namespace dealii::Utilities;
```

Comment:
Is there a reason you are removing the "using namespace dealii" line? This should stay here I think.